server/routers/jobs_router.py

```
@@ -684,18 +655,47 @@ async def analyze_job_direct(
+            "job": {"id": job_data.id},
             "analysis": evaluation
         }
     except Exception as e:
         logger.error(f"Erreur critique analyse directe : {str(e)}")
         raise HTTPException(status_code=500, detail=str(e))
+
+
+@router.get("/offer/{job_id}", summary="Get job offer details by ID")
+async def get_job_offer(
+    job_id: str,
+    current_user: User = Depends(get_current_user)
```

Ruff (0.14.13)

Unused function argument: `current_user`

(ARG001)